ioc/Config/Jail/JailConfig.py

```diff
@@ -37,7 +37,7 @@ class JailConfig(ioc.Config.Jail.BaseConfig.BaseConfig):
     legacy: bool = False
     jail: typing.Optional['ioc.Jail.JailGenerator']
     data: dict = {}
-    ignore_host_defaults: bool = False
+    ignore_user_defaults: bool
```

Comment:
i just realized, that new name is quite…
it could be confused with ignoring `user.*` settings.

Reply:
Indeed! I'm glad you've mentioned it. Is `ignore_source_config` maybe a better name?

There are three layers of configuration
1. Jail Config (`config.json`, `ZFS properties`, ...)
2. Source Config (`/iocage/defaults.json`, ...)
3. Hardcoded Defaults